src/Poseidon/Janno.hs

```diff
@@ -20,36 +19,39 @@ module Poseidon.Janno (
     createMinimalJanno
 ) where
 
-import           Poseidon.Utils             (PoseidonException (..), renderPoseidonException)
+import           Poseidon.Utils             (PoseidonException (..),
+                                             renderPoseidonException)
 
 
 import           Control.Applicative        (empty, optional)
-import           Data.Bifunctor             (second)
 import           Control.Exception          (throwIO, try)
-import           Control.Monad              (when, unless)
+import           Control.Monad              (unless, when)
+import           Data.Aeson                 (FromJSON, Options (..), ToJSON, Value(..),
+                                             defaultOptions, genericToEncoding, toJSON, parseJSON,
+                                             toEncoding)
+import           Data.Bifunctor             (second)
 import qualified Data.ByteString.Char8      as Bchs
 import qualified Data.ByteString.Lazy.Char8 as Bch
 import           Data.Char                  (ord)
 import qualified Data.Csv                   as Csv
-import qualified Data.HashMap.Lazy          as HM
-import           Data.Either                (isRight, isLeft, rights, lefts)
+import           Data.Either                (isLeft, isRight, lefts, rights)
```

Comment:
Really? :laughing: 
Ah wait - your styler did this automatically?

Reply:
Yeah, that's just "stylish-haskell" in vscode.

Reply:
I just can't get this to work on my systems. Maybe I should investigate once more